featuretools/tests/primitive_tests/test_features_deserializer.py

```diff
@@ -11,7 +11,7 @@
 
 
 def test_single_feature(es):
-    feature = ft.IdentityFeature(es['log']['value'])
+    feature = ft.IdentityFeature(es, 'log', 'value')
```

Comment:
In some files we are using `ft.IdentifyFeature` and in others we import `IdentifyFeature` and use it. Not sure if there is a good reason for that, but I just noticed it in case we want to clean up and be consistent in the tests.

Reply:
we're also very inconsistent on logical types, either using `ww.logical_types.Boolean`, `ltypes.Boolean`, or `Boolean`

Reply:
This doesn't need to hold up anything here - maybe we should create a separate issue and clean that up later?

Reply:
Created #1510  to track the woodwork issue

For the featuretools imports, thoughts on the stance that we should avoid the `ft` import unless we are trying to test that we can access the class or method from the `ft` namespace?

Reply:
I have no issues with that stance, and I think it will make the code cleaner without `ft.` sprinkled about everywhere...

Reply:
ceated #1511 to track the `ft.` use in tests issue